tests/e2e/e2e_groups_test.go

```diff
@@ -162,16 +159,18 @@ func (s *IntegrationTestSuite) prepareGroupFiles(c *chain, adminAddr string, mem
 			Metadata: "Bob",
 		},
 	}
+
 	s.writeGroupMembers(c, members, originalMembersFilename)
 
-	newMembers := append(members, GroupMember{
+	newMembers := append(members, group.MemberRequest{
 		Address:  member3Address,
 		Weight:   "1",
 		Metadata: "Charlie",
 	})
+
 	s.writeGroupMembers(c, newMembers, addMemberFilename)
 
-	removeMembers := append(members, GroupMember{
+	removeMembers := append(members, group.MemberRequest{
```

Comment:
can you just directly change the `weight` , this `append` way remove members require the member to be removed the last member in the slice.

Reply:
sure thing